include/mbgl/style/expression/image.hpp

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <mbgl/style/conversion.hpp>
+#include <mbgl/util/color.hpp>
+#include <mbgl/util/optional.hpp>
+
+#include <string>
+#include <vector>
+
+namespace mbgl {
+namespace style {
+namespace expression {
+
+class Image {
+public:
+    Image() = default;
+    Image(const char* imageID);
+    Image(std::string imageID);
+    explicit Image(std::string imageID, bool available);
+    bool operator==(const Image&) const;
+    mbgl::Value serialize() const;
```

Comment:
"serialize" is more for text or row data, maybe `toValue`?

Reply:
renamed toValue